test/unit/controllers/declaration/ProcedureCodesControllerSpec.scala

```diff
@@ -231,7 +234,9 @@ class ProcedureCodesControllerSpec extends ControllerSpec with ErrorHandlerMocks
 
         val result = controller.submitProcedureCodes(itemId)(postRequestAsFormUrlEncoded(correctForm: _*))
 
-        status(result) mustBe SEE_OTHER
+        await(result) mustBe aRedirectToTheNextPage
+        thePageNavigatedTo.toString mustBe "/items/itemId12345/fiscal-information"
```

Comment:
Noooo use `controllers.declaration.routes.FiscalInformationController.displayPage()` without the `.toString`

Means we can change routes without breaking stuff

Reply:
Similar for the other hardcoded URLs

Reply:
This has been addressed.